app/views/shared/_footer.html.erb

```diff
@@ -0,0 +1,10 @@
+<footer class="footer wrapper">
+  <p>Made for <a href="https://hacktoberfest.digitalocean.com/">Hacktoberfest 2017</a>.</p>
+  <p>Want to build a site like this? <%= link_to 'Check out the source', 'https://github.com/yogodoshi/cansei' %>.</p>
+
+  <nav class="footer-social-links">
```

Comment:
Wdyt of merging this with the line above? Is it really better to have 2 different things/lines for the same reason? 🤔

Reply:
I think they denote different things. Even though we recognize the :octocat:, there are probably plenty of users who don't. 

Also, I think if I were a random user I wouldn't think that this site was easily cloneable (and was designed to be!), even with the Github logo. Let me know what you think.

Reply:
> Even though we recognize the :octocat:, there are probably plenty of users who don't.

Agreed!

That's why I wanted to remove the github icon. Cause the "want to  build a site like this?" works for both kind of users. When I said "merging" I wanted to say that if you want to you could simply add the github icon to the "want to build a site like this?" line :)

I just think we don't need 2 lines for the same reason: letting people know they can easily create a site like this and the github icon, which to me, denotes the same thing but for developers. Yeah, the phrase could mean a free/paid tool/service but just clicking or even hovering the link the user would see a github link there. And there's also the possibility of adding the icon to beginning or end of the phrase if you want :)

Reply:
Ahh, well I also imagined that people may want to hookup social media accounts like twitter, or facebook groups at some point. That's why I called this area `footer-social-links`. It's all good! I noticed you removed them anyway 👍